Jenkinsfile

```diff
@@ -248,20 +259,19 @@ pipeline {
 
                             steps {
                                 timeout(time: longTimeoutMinutes, unit: 'MINUTES') {
-                                    unstash name: "zeebe-build"
                                     runMavenContainerCommand('.ci/scripts/distribution/it-java.sh')
                                 }
                             }
 
                             post {
                                 always {
                                     junit testResults: "**/*/TEST*${SUREFIRE_REPORT_NAME_SUFFIX}*.xml", keepLongStdio: true
+                                    stash allowEmpty: true, name: itFlakyTestStashName, includes: '**/FlakyTests.txt'
```

Comment:
Moved here to avoid error if there are only flaky tests in the main agent, since you cannot test if a stash is present or not